README.md

```diff
@@ -41,27 +41,43 @@ Tested on Ubuntu 20 *WSL* and Centos 7
 
 <br>
 
- ## Usage:
+# Usage:
+
+ ### Common Arguments
+
+ | Usage: | `./bin/run.sh <command> [OPTION]...` |
+ |---|---|
+ | `-v` | Verbose logging (Info level) |
+ | `-vv` | Verbose logging (Debug level) |
+ | `-vvv` | Verbose logging (Trace level) |
+ | `--quiet` | Suppress logging to console, except for errors |
 
 ### Scaffold Importance Sampling
 
-| Usage: | `/bin/run.sh simulate [OPTION]...` |
+| Usage: | `./bin/run.sh simulate [OPTION]...` |
 |-|-|
 | `-M` | Transition multiplier (default: 2) |
 | `-Tmax` | Maximum transitions before truncating (default: 1,000) |
 | `--Nruns` | Number of stochastic runs (default: 100,000) |
 | `--raw` | Print raw output values, no labels |
 | `--model` | Prism model file name |
 
-### Sparse Model Generator
-| Usage: | `./bin/run.sh generate [OPTION]...` |
+### Model Generator (MGen)
+| Usage: | `./bin/run.sh mgen <mgenID> [OPTION]...` |
 |-|-|
+| `mgenID` | The model generator to use (currently only `sparse`) |
 | `-I` | Number of models to generate, iterations. (default: 1) |
 | `-N` | Number of states to generate (default: 10) |
 | `--target-state` | Index of the target state, zero indexed (default: `numberOfState`-1)
 | `--output` | Name of the output Prism file (default: `sparse-model-%i%.pm`)
 | `--config` | Model generator json config file. See example below
 
+<br>
+
+The absolute path to `/.bin/run.sh` could be added to the user or system path
+as a symlink `imsam` if desired. Later version may have an install option to
```

Comment:
This can be done quite easily using `ln -T /usr/local/bin/imsam ./bin/run.sh`. Consider adding this command to the docs so users can copy/paste it rather than having to formulate it themselves.